api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,17 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+    /**
+	 * Gets all conditions associated with a patient
```

Comment:
Can you make it explicit in the documentation that this also includes inactive conditions?

Reply:
@dkayiwa  I updated the PR :)